site-scrapers/CVS.js

```diff
@@ -0,0 +1,43 @@
+const sites = require("../data/sites.json");
+
+const siteName = "CVS";
+const site = sites[siteName];
+
+module.exports = async function GetAvailableAppointments(browser) {
+    console.log(`${siteName} starting.`);
+    const webData = await ScrapeWebsiteData(browser);
+    console.log(`${siteName} done.`);
+    return webData.map((responseLocation) => {
+        let hasAvailability = parseInt(responseLocation.totalAvailable)
+            ? true
+            : false;
+        let availability = responseLocation.totalAvailable;
+        return {
+            name: `${siteName} (${responseLocation.city})`,
```

Comment:
Is it too much to ask for the city names in Title Case, cf. https://stackoverflow.com/a/4878800?
`      name: 'CVS (NEW BEDFORD)'`
isn't the best.

Reply:
Added this code, thanks for the link.